### ISSUE TITLE
Not enough arguments. - org.multimc.NotFoundException

### ISSUE THREAD

**Issue by phoe** [MultiMC/MultiMC5#2092]
I have copied `~/.local/share/multimc/` from one Linux instance on my laptop to the other.
When trying to launch Minecraft:

```
MultiMC version: 0.6.0-1069


Minecraft folder is:
/home/phoe/.local/share/multimc/instances/1.12.2/minecraft


Java path is:
/usr/bin/java


Java is version 9.0.1, using 64-bit architecture.


Intel(R) Core(TM) i3-4100M CPU @ 2.50GHz
Intel Corporation 4th Gen Core Processor Integrated Graphics Controller (rev 06)

Subsystem: Lenovo 4th Gen Core Processor Integrated Graphics Controller

Kernel driver in use: i915

OpenGL version string: 3.0 Mesa 17.3.1

Main Class:
  

Native path:
  /home/phoe/.local/share/multimc/instances/1.12.2/natives

Libraries:

Native libraries:

Params:
  

Window size: max (if available)

Java Arguments:
[-Xms512m, -Xmx1024m, -Duser.language=en]


Minecraft process ID: 4735


Using onesix launcher.

Not enough arguments.
org.multimc.NotFoundException
	at org.multimc.ParamBucket.all(ParamBucket.java:43)
	at org.multimc.onesix.OneSixLauncher.processParams(OneSixLauncher.java:55)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:212)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Exiting with -1
Process exited with code 255.
Clipboard copy at: 03 Jan 2018 23:42:48 +0100
```

It seems like https://github.com/MultiMC/MultiMC5/blob/12c3683ec07788bf4be03221b0232b4eae27caba/libraries/launcher/org/multimc/ParamBucket.java#L43 could use throwing `NotFoundException` with the offending key, because right now this exception provides no debugging information.

**Comment by phoe** [comment]
Also, when trying to pick Minecraft version:
```
List update failed:
Job 'Download of meta file net.minecraft/index.json' failed to process:
https://v1.meta.multimc.org/net.minecraft/index.json
```

**Comment by phoe** [comment]
Also when updating:
```
Failed to fetch changelog... Error: Job 'Changelog' failed to process: https://raw.githubusercontent.com/MultiMC/MultiMC5/stable/changelog.md
```

**Comment by phoe** [comment]
Console logs:
```
└─▪ /opt/multimc/run.sh 
MultiMC Dir: /home/phoe/.local/share/multimc
No missing dependencies found.
    0.028 C "Failed to read the account list file (accounts.json)."
    0.057 W QMetaObject::connectSlotsByName: No matching signal for on_actionInstanceSettings_triggered()
    0.112 W QSslSocket: cannot resolve CRYPTO_num_locks
    0.112 W QSslSocket: cannot resolve CRYPTO_set_id_callback
    0.112 W QSslSocket: cannot resolve CRYPTO_set_locking_callback
    0.112 W QSslSocket: cannot resolve ERR_free_strings
    0.112 W QSslSocket: cannot resolve sk_new_null
    0.112 W QSslSocket: cannot resolve sk_push
    0.112 W QSslSocket: cannot resolve sk_free
    0.112 W QSslSocket: cannot resolve sk_num
    0.112 W QSslSocket: cannot resolve sk_pop_free
    0.112 W QSslSocket: cannot resolve sk_value
    0.112 W QSslSocket: cannot resolve SSL_library_init
    0.112 W QSslSocket: cannot resolve SSL_load_error_strings
    0.112 W QSslSocket: cannot resolve SSLv3_client_method
    0.112 W QSslSocket: cannot resolve SSLv23_client_method
    0.112 W QSslSocket: cannot resolve SSLv3_server_method
    0.112 W QSslSocket: cannot resolve SSLv23_server_method
    0.112 W QSslSocket: cannot resolve X509_STORE_CTX_get_chain
    0.112 W QSslSocket: cannot resolve OPENSSL_add_all_algorithms_noconf
    0.112 W QSslSocket: cannot resolve OPENSSL_add_all_algorithms_conf
    0.112 W QSslSocket: cannot resolve SSLeay
    0.112 W QSslSocket: cannot resolve SSLeay_version
    0.113 W QSslSocket: cannot call unresolved function CRYPTO_num_locks
    0.113 W QSslSocket: cannot call unresolved function CRYPTO_set_id_callback
    0.113 W QSslSocket: cannot call unresolved function CRYPTO_set_locking_callback
    0.113 W QSslSocket: cannot call unresolved function SSL_library_init
    0.113 W QSslSocket: cannot call unresolved function SSLv23_client_method
    0.114 W QSslSocket: cannot call unresolved function sk_num
    1.269 W QSslSocket: cannot call unresolved function SSLv23_client_method
    1.270 W QSslSocket: cannot call unresolved function SSL_library_init
    1.270 W We got a connection error when networkLayerState is Unknown
    1.270 C Failed  "https://multimc.org/rss.xml"  with reason  99
    1.305 W QSslSocket: cannot call unresolved function SSLv23_client_method
    1.305 W QSslSocket: cannot call unresolved function SSL_library_init
    1.330 W QSslSocket: cannot call unresolved function SSLv23_client_method
    1.330 W QSslSocket: cannot call unresolved function SSL_library_init
    1.331 C Failed  "https://multimc.org/rss.xml"  with reason  99
    1.371 W QSslSocket: cannot call unresolved function SSLv23_client_method
    1.371 W QSslSocket: cannot call unresolved function SSL_library_init
    1.371 C Failed  "https://multimc.org/rss.xml"  with reason  99
    1.397 W QSslSocket: cannot call unresolved function SSLv23_client_method
    1.397 W QSslSocket: cannot call unresolved function SSL_library_init
    1.397 C Failed  "https://multimc.org/rss.xml"  with reason  99
    1.398 C Task "NetJob(News RSS Feed)" failed:  "Job 'News RSS Feed' failed to process:
https://multimc.org/rss.xml"
```

**Comment by phoe** [comment]
Issue resolved via https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3001#issuecomment-309673453

TL;DR: `apt install libssl1.0-dev`
